rotator_indi.cpp

```diff
@@ -0,0 +1,369 @@
+/*
+*  rotator_indi.cpp
+*  PHD Guiding
+*
+*  Created by Philipp Weber
+*  Copyright (c) 2023 Philipp Weber
+*  All rights reserved.
+*
+*  This source code is distributed under the following "BSD" license
+*  Redistribution and use in source and binary forms, with or without
+*  modification, are permitted provided that the following conditions are met:
+*    Redistributions of source code must retain the above copyright notice,
+*     this list of conditions and the following disclaimer.
+*    Redistributions in binary form must reproduce the above copyright notice,
+*     this list of conditions and the following disclaimer in the
+*     documentation and/or other materials provided with the distribution.
+*    Neither the name of Craig Stark, Stark Labs nor the names of its
```

Comment:
Some of the older source files that came from the original Stark Labs PHD (PHD1) contained this copyright referencing to Stark Labs.   In new code that is not from Stark Labs we no longer reference Stark labs in the copyright notice (though in some cases like this the copyright got copy-pasted inadvertently)
Our preference would be to use the copyright notice referring only to openphdguiding.org -- you can copy the text from scope_manual_pointing.h.